app/src/ui/lib/text-box.tsx

```diff
@@ -271,6 +274,14 @@ export class TextBox extends React.Component<ITextBoxProps, ITextBoxState> {
           this.inputElement.blur()
         }
       }
+    } else if (
+      this.props.type === 'search' &&
+      event.key === 'Enter' &&
+      value !== undefined &&
```

Comment:
Should this be `typeof value !== 'undefined'` or is that not a TypeScript thing?

Reply:
Not a TypeScript expert, but I think that's not needed and `value !== undefined` is just fine 😄